raiden/network/rpc/client.py

```diff
@@ -270,6 +267,21 @@ def set_verbosity(self, level):
     def block_number(self):
         return self.client.blocknumber()
 
+    def is_synced(self):
+        result = self.client.call('eth_syncing')
```

Comment:
I am afraid that this is not enough to determine if we are syncing.

`eth_syncing` will be returning `false` even if we are not synced but if the syncing process has not yet started. And since some times the syncing process may take even minutes to start then this is not the best way to detect if we are synced.

A better way would be to use some oracle of the latest block and compare if we are close to it, let's say 5 blocks from it.

The simplest oracle I can imagine would be etherscan, or we could have our own dedicated server. For etherscan that would be the following API call:

`https://ropsten.etherscan.io/api?module=proxy&action=eth_blockNumber`

Reply:
Humm, how can we tell which BC the eth node is connected to?

Reply:
Good question. We can't. Best way would be to let the user specify it themselves if we want to know it on the raiden side.

Reply:
Once web3 1.0 is "released" it will be: https://web3js.readthedocs.io/en/1.0/web3-eth-net.html

Reply:
Currently, there's the [web3.version.network](https://github.com/ethereum/wiki/wiki/JavaScript-API#web3versionnetwork). Isn't it enough?

Reply:
`web3.version.network` should be good enough. It's the `chainID`. Did not know that call was available from web3.js.